packages/react-components/src/components/styles/UnsupportedEnvironment.styles.tsx

```diff
@@ -58,3 +59,29 @@ export const iconStyles: IIconStyles = {
     paddingBottom: '1rem'
   }
 };
+
+/**
+ * @internal
+ */
+export const continueAnywayButtonStyles = (theme: Theme): IButtonStyles => {
```

Comment:
Do you have a link to the figma designs? [These](https://www.figma.com/file/UL3EWh73Jut5PH6jyAjWsy/Call-Readiness?node-id=1093%3A55556&t=j1OqArSwn4Oa6x8v-0) look the same as the standard fluent default button - looking to understand what differs from the regular button

Reply:
the regular button is blue, these styles will change when Juan gets the figma updated.

Reply:
PrimaryButton is blue, DefaultButton is white https://developer.microsoft.com/en-us/fluentui#/controls/web/button

Reply:
ah right I cannot read, will update it and see what that looks like